<?xml version="1.0"?>
<launch>
  <node pkg="laser_line_extraction" type="line_extraction_node" name="line_extractor">
    <param name="~frequency" value="30.0" />
    <!-- <param name="~frame_id" value="hokuyo_link" />
    <param name="~scan_topic" value="hokuyo_scan" /> -->
    <!-- <param name="~frame_id" value="ydlidar_front_link" />
    <param name="~scan_topic" value="scan_front" /> -->
    <param name="~frame_id" value="base_link" />
    <param name="~scan_topic" value="scan" />
    <param name="~publish_markers" value="true" />
    <param name="~bearing_std_dev" value="1e-5" />
    <param name="~range_std_dev" value="0.012" />
    <param name="~least_sq_angle_thresh" value="0.0001" />
    <param name="~least_sq_radius_thresh" value="0.0001" />
    <param name="~max_line_gap" value="0.4" />
    <param name="~min_line_length" value="0.1" />
    <param name="~min_range" value="0.03" />
    <param name="~max_range" value="10.0" />
    <param name="~min_split_dist" value="0.05" />
    <param name="~outlier_dist" value="0.05" />
    <param name="~min_line_points" value="4" />
  </node>
</launch>